tests/test_datasets.py

```diff
@@ -58,12 +61,9 @@ def test_datasets_with_dataset_id(test_input):
 
 @pytest.mark.parametrize("test_input", [
     "latest",
-    DATASET_ID,
 ])
-def test_datasets_zip_with_makes_generated_request(test_input, mocker):
-    mocker.patch('urllib.request.urlopen')
-    mocker.patch('lcboapi.LCBOAPI.generate_api_request')
-    api.datasets_zip(test_input)
-    # pylint: disable=E1101
-    api.generate_api_request.assert_called_once_with(
-        'datasets/{}.zip'.format(test_input))
```

Comment:
Please bring this test back and refactor it to work again

Reply:
Actually, that no longer makes sense to do. Let's refactor the `generate_api_request` method to be private again and be done with it.